backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/specific/AbstractSelectFilter.java

```diff
@@ -40,11 +33,6 @@
 	 */
 	protected BiMap<String, String> labels = ImmutableBiMap.of();
 
-	@JsonIgnore
-	protected Set<String> values = new HashSet<>();
-	@JsonIgnore
-	protected transient QuickSearch<FilterSearchItem> sourceSearch;
-
 	@JsonIgnore
 	private final int maximumSize;
```

Comment:
Wir der Upgrade zum BIG_MULTI noch gebraucht? eigentlich doch schon oder?

Reply:
> Wir der Upgrade zum BIG_MULTI noch gebraucht? eigentlich doch schon oder?

Da es relativ kompliziert ist das in der aktuellen Struktur umzusetzen, habe ich ihn rausgenommen und es dokumentiert. Ich liefere dann noch Konzeptanpassungen nach wo sie fehlen.